src/components/download/mod.rs

```diff
@@ -38,41 +38,44 @@ impl AccountIdContext {
                     };
 
                     let input_args = serde_json::to_string(&crate::socialdb_types::SocialDbQuery {
-                        keys: vec![format!("{account_id}/widget/**")],
+                        keys: vec![format!("{account_id}/widget/*")],
                     })
                     .wrap_err("Internal error: could not serialize SocialDB input args")?;
 
                     let call_result = network_config
                         .json_rpc_client()
                         .blocking_call_view_function(
                             near_social_account_id,
-                            "get",
+                            "keys",
                             input_args.into_bytes(),
                             near_primitives::types::Finality::Final.into(),
                         )
                         .wrap_err("Failed to fetch the components state from SocialDB")?;
+                    let keys: SocialDbKeys = serde_json::from_slice(&call_result.result)
+                        .wrap_err("Failed to parse the components state from SocialDB")?;
```

Comment:
I think this should work, right?

```suggestion
                    let keys: SocialDbKeys = call_result.parse_result_from_json()?;
```